src/_articles/language/mixins.md

```diff
@@ -23,10 +23,10 @@ Support for mixins changed in Dart 1.13, and it will again in Dart 2:
   specifically, it is behind the `--supermixin` flag in the command-line
   analyzer. It is also available in the analysis server, behind a
   client-configurable option. Dart2js and dartdevc do not support super mixins.
-- In Dart 2, mixins are expected to have fewer restrictions. For example,
+- In Dart 2.1, mixins are expected to have fewer restrictions. For example,
   Flutter supports mixins calling `super()` and extending from a class other
   than `Object`, but the syntax is expected to change before appearing in all
-  Dart SDKs. For details, see the [proposed mixin specification.][Dart 2.0 Mixins]
+  Dart SDKs. For details, see the [mixin specification.][Dart 2.1 Mixins]
```

Comment:
It looks like you need to update the place where the URL is specified ([Dart 2.0 Mixins] -> [Dart 2.1 Mixins], and probably change the URL to the new one: https://github.com/dart-lang/language/blob/master/working/0006.%20Super-invocations%20in%20mixins/0007.%20Mixin%20declarations/lrhn-strawman.md).